static/app/components/events/eventTagsAndScreenshot/tags.tsx

```diff
@@ -24,68 +24,71 @@ type Props = {
   projectSlug: Project['slug'];
 };
 
-function Tags({event, projectSlug}: Props) {
-  const sentryTags = getSentryDefaultTags();
+export const EventTagsDataSection = forwardRef<HTMLElement, Props>(
```

Comment:
Forwarding the ref since `Highlights` uses it and we don't want wrappers in `<DefaultGroupEventDetails />`